api/src/main/java/org/pmiops/workbench/cohorts/CohortService.java

```diff
@@ -1,19 +1,28 @@
 package org.pmiops.workbench.cohorts;
 
 import java.util.List;
+import java.util.stream.Collectors;
 import org.pmiops.workbench.db.dao.CohortDao;
 import org.pmiops.workbench.db.model.DbCohort;
+import org.pmiops.workbench.model.Cohort;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class CohortService {
-  private CohortDao cohortDao;
+  private final CohortDao cohortDao;
+  private final CohortMapper cohortMapper;
 
-  public CohortService(CohortDao cohortDao) {
+  @Autowired
+  public CohortService(CohortDao cohortDao, CohortMapper cohortMapper) {
     this.cohortDao = cohortDao;
+    this.cohortMapper = cohortMapper;
   }
 
-  public List<DbCohort> findAll(List<Long> cohortIds) {
-    return (List<DbCohort>) cohortDao.findAll(cohortIds);
+  public List<Cohort> findAll(List<Long> cohortIds) {
+    return ((List<DbCohort>) cohortDao.findAll(cohortIds))
+        .stream()
+            .map(dbCohort -> cohortMapper.dbModelToClient(dbCohort))
```

Comment:
Can this be .map(cohortMapper::dbModelToClient)

Reply:
Done.